docker-compose.yml

```diff
@@ -8,30 +8,25 @@ secrets:
     file: ./src/secrets/quote.txt
 
 services:
-  example:
+  pca-report-library:
     # Run the container normally
     build:
       # VERSION must be specified on the command line:
       # e.g., --build-arg VERSION=0.0.1
       context: .
       dockerfile: Dockerfile
-    image: cisagov/example
+    image: cisagov/pca-report-library
     init: true
     restart: "no"
     environment:
       - ECHO_MESSAGE=Hello World from docker-compose!
-    ports:
-      - target: 8080
-        published: 8080
-        protocol: tcp
-        mode: host
     secrets:
       - source: quote_txt
         target: quote.txt
 
-  example-version:
+  pca-report-library-version:
     # Run the container to collect version information
-    image: cisagov/example
+    image: cisagov/pca-report-library
     init: true
     restart: "no"
-    command: --version
+    command: pca-report-generator --version
```

Comment:
If you make `pca-report-generator` your `ENTRYPOINT`, then you would only need to specify `--version` here.

Reply:
If pca-report-generator is the entrypoint, wouldn't pca-report-compiler and pca-report-templates not work?

Reply:
See https://github.com/cisagov/pca-report-generator-docker/pull/10#discussion_r746977908 for info about this.

Reply:
Resolved in 120e86be18d2c721e228a9efcae71148b6820d03